src/MemoryTracing/MemoryTracingUtils.cpp

```diff
@@ -24,18 +24,12 @@ using orbit_grpc_protos::kMissingInfo;
 using orbit_grpc_protos::ProcessMemoryUsage;
 using orbit_grpc_protos::SystemMemoryUsage;
 
-SystemMemoryUsage ParseMemInfo(std::string_view meminfo_content) {
-  SystemMemoryUsage memory_info;
-  memory_info.set_total_kb(kMissingInfo);
-  memory_info.set_free_kb(kMissingInfo);
-  memory_info.set_available_kb(kMissingInfo);
-  memory_info.set_buffers_kb(kMissingInfo);
-  memory_info.set_cached_kb(kMissingInfo);
-
-  if (meminfo_content.empty()) return memory_info;
+void GetValuesFromMemInfo(std::string_view meminfo_content,
+                          SystemMemoryUsage* system_memory_usage) {
+  if (meminfo_content.empty()) return;
```

Comment:
Shouldn't in this case reset the values from system_memory_usage to "kMissingInfo"?

Reply:
Thanks for the suggestion. I moved the resetting of system_memory_usage fields to the corresponding Get method.